simple.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+let input = FileHandle.standardInput
+
+if let text = String(data: input.readDataToEndOfFile(), encoding: .utf8) {
+
+  let words = text.split { $0.isWhitespace || $0.isNewline }
+
+  var counts = [String:Int]()
+
+  for word in words {
+    let canon = word.lowercased()
+    counts[canon, default: 0] += 1
+  }
+
+  let ordered = counts.sorted { $0.1 > $1.1 }
+
+  ordered.forEach { entry in
```

Comment:
Why not `for entry in ordered {`?

Reply:
I like to use forEach when nothing is mutated in the loop.